### PR TITLE
[ADDED] natsConnection_GetClientIP()

### PR DESCRIPTION
**Pull request by kozlovic**
Returns the IP as determined by the server when accepting this
connection.

Signed-off-by: Ivan Kozlovic <ivan@synadia.com>

### CODE REVIEW

File: src/conn.c [hunk 1]
```diff
@@ -525,6 +526,8 @@ _processInfo(natsConnection *nc, char *info, int len)
         s = nats_JSONGetULong(json, "client_id", &(nc->info.CID));
     if (s == NATS_OK)
         s = nats_JSONGetStr(json, "nonce", &(nc->info.nonce));
+    if (s == NATS_OK)
+        s = nats_JSONGetStr(json, "client_ip", &(nc->info.clientIP));
 
 #if 0
```
Comment (derekcollison) [inline src/conn.c:532]: You leave these in?

Comment (kozlovic) [reply src/conn.c:532]: Sorry, not sure I understand the question

Comment (kozlovic) [reply src/conn.c:532]: The #if 0 you mean?

Comment (derekcollison) [reply src/conn.c:532]: Yes, common to leave those in for folks to easily debug stuff?

Comment (derekcollison) [reply src/conn.c:532]: Been so long since I have done C ;)

Comment (kozlovic) [reply src/conn.c:532]: That was on old vestige of debugging the parsing of the INFO. I will remove.

**Pull request merged**